account_reports_improved/views/account_financial_report_line.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+        <record id="view_account_financial_report_line_form" model="ir.ui.view">
+            <field name="name">account.financial.html.report.line.form</field>
+            <field name="model">account.financial.html.report.line</field>
+            <field name="inherit_id" ref="account_reports.view_account_financial_report_line_form"/>
+            <field name="arch" type="xml">
+                <form position="replace">
```

Comment:
@dufresnedavid If the objective is to replace the form completely it would be better to declare a new one without inherit and change the action.

Reply:
@joaoalf excellent idea, I didn't think about that